src/components/App/index.js

```diff
@@ -16,7 +16,7 @@
 
 import React from 'react';
 import { Provider } from 'react-redux';
-import { BrowserRouter, Route } from 'react-router-dom';
+import { HashRouter as Router, Route } from 'react-router-dom';
```

Comment:
I like the cleanliness of not using a hash based URL but its probably going to be problematic for folks that want to embed this in other servers like Spring Boot.  Maybe switching to a hash based router isn't necessary so maybe @tacigar can provide some clarity if there is a better approach.